roofit/CMakeLists.txt

```diff
@@ -13,3 +13,4 @@ add_subdirectory(roostats)
 if(xml)
   add_subdirectory(histfactory)
 endif()
+add_subdirectory(hs3)
```

Comment:
Does this depend on a specific package being there? If yes, you can include it in `if(<package>)`.
It probably also makes sense to add a switch to the ROOT build options like `-Dhs3=ON/OFF` to enable/disable YAML.

I also just realised that this creates a broken build configuration. This commits declares a subdirectory, but it doesn't add it. ROOT will not build if somebody checks out this commit. It's probably safe if this commit is merged into the third commit.

Reply:
One more thing:
One could also disallow `hs3=ON` if no YAML has been found. Need to figure out with the CMake experts how to check if e.g. RapidYAML is available.

Reply:
The way it's configured right now is that it uses RapidYAML if found, and the ROOT-included nlohmann JSON Parser otherwise. Both provide the JSON functionality, but only RapidYAML will read and write YAML files.

Reply:
@cburgard @hageboeck sorry for delay with answer, I can help you with suggestions for CMake. Currently I don't see a lot of CMake code, including ```hs3``` option, find procedure for https://github.com/biojppm/rapidyaml  Do I missing something?

Reply:
It seems I forgot to add the actual CMakeLists file for the hs3 subdirectory. It's added now. Please take a look!